### DISCUSSION TITLE
Issue installing on older Laptop

### DISCUSSION THREAD

**Discussion by talonmyburgh** [artisan-roaster-scope/artisan#1240] [Q&A]
**Describe the bug**
For our artisan roastery in South Africa, we picked up a small Chomebook to use artisan and some Phidget devices to read from the Roaster. Knowing that ChromeOS would not run artisan, I installed an OS called [Gallium](https://galliumos.org/). This works well and currently installed is a linux instance with kernel version 4.16.18.

I cannot install the latest version due to the Glibc version upgrade recently made (the system has Ubuntu GLIBC 2.27-3ubuntu1.6). 

**To Reproduce**
To try to install from source, I have downloaded the artisan-1.6.2 release and run:
`sudo bash ./artisan-1.6.2/src/build-linux.sh`

**Expected behavior**
I initially had some PyQT errors where for Python2.7 I needed to clone some unofficial PyQT5 repo but still had no luck. There error I now am stuck with is:
`cp: cannot stat '/home/<user>/.local/lib/python3.6/site-packages/matplotlib/mpl-data' : No such file or directory`

Towards the start of the installer trace, there are `export` calls to choose that python3's site-packages path for PYTHON_PATH and I'm not sure how to change that.

**Setup**
 - Artisan Version:  v1.6.2:
 - Artisan Build: d2039bf
 - Platform: Linux kernel 4.16.18

**Additional context**
An ideal outcome is if some of the releases pre major changes could contain build (packaged) installs so that one does not need to install from source. Otherwise, if someone has a way to send me a `.deb` for this system that would be very helpful.

**Comment by roasterdave** [comment]
Perhaps someone in the community will be able to help.

The Artisan development team does not have the resources to support older releases (v1.6.2 is over four years old).  Nor can we provide support for platforms beyond the [Supported Platforms](https://github.com/artisan-roaster-scope/artisan/releases/latest) list.  For this and many other reasons we recommend running the latest version of Artisan.

If you, or anyone, succeed with running the latest Artisan version on a Chromebook please share back with the community.  

`Since this request is not a bug it will be converted from an Issue to a Discussion. `

**Reply by talonmyburgh** [reply]
Thanks @roasterdave. I do understand that support is not available but is it possible to have build images available for milestone releases even if they are from quite a while back? In this way you do not require that users have to build their software from source and encounter the issues I am encountering?
It is prohibitive to require this of users because they would like to use an earlier version of the software.

**Reply by roasterdave** [reply]
I'm sorry, those old builds are no longer available.   Can you run directly from the source?
